modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -205,17 +225,49 @@ def prepare_shared_files(self, local_dir):
         self._runcmd(f"chmod a+x {local_dir}/*.sh")
         self._runcmd(f"{local_dir}/mkcerts.sh")
 
+    def final_jvm_opts(self):
```

Comment:
May be just 
```
@property
def jvm_opts(self):
```

Reply:
Also, it is good idea to avoid getters-setters and use @property and @property.setter.

Reply:
All these params and functions should be eliminated at all. Very hard to read and understand the code